packages/docs/docs/docs/linking.md

```diff
@@ -1,127 +1,281 @@
+
+```console
+mkdir token-exchange && cd token-exchange
+npm init -y
+npm install --save-dev zos
+npx zos init
+```
+
+> Note: The full code for this project is available in our [Github repo](https://github.com/zeppelinos/zos/tree/v2.4.0/examples/linking-contracts).
+
+## Linking the contracts EVM package
+
+We will first get ourselves an ERC20 token. Instead of coding one from scratch, we will use the one provided by the [OpenZeppelin contracts EVM package](https://github.com/OpenZeppelin/openzeppelin-eth). An EVM package is a set of contracts set up to be easily included in a ZeppelinOS project, with the added bonus that the contracts' code _is already deployed in the Ethereum network_. This is a more secure code distribution mechanism, and also helps you save gas upon deployment.
+
+<!-- TODO: Add guide on what is an EVM package behind the scenes, instead of linking to a blogpost -->
+
+> Note: Check out [this article](https://blog.zeppelinos.org/open-source-collaboration-in-the-blockchain-era-evm-packages/)to learn more about EVM packages.
+
+To link the OpenZeppelin contracts EVM package into your project, simply run the following:
+
+```console
+npx zos link openzeppelin-eth@2.2.0
+```
+
+This command will download the EVM package (bundled as a regular npm package), and connect it to your ZeppelinOS project. We now have all of OpenZeppelin contracts at our disposal, so let's create an ERC20 token!
+
+> Note: Make sure you install `openzeppelin-eth` and not `openzeppelin-solidity`. The latter is set up for general usage, while `openzeppelin-eth` is tailored for being used with ZeppelinOS. This means that its contracts are [already set up to be upgradeable](writing_contracts#use-upgradeable-packages).
+
+## Creating an ERC20 token
+
+Let's deploy an ERC20 token contract to our development network. But first, make sure to install ganache if needed, and start a new instance:
+
+```console
+npm install -g ganache-cli
+ganache-cli --deterministic
+```
+
+For setting up the token, we will be using the [StandaloneERC20 implementation](https://github.com/OpenZeppelin/openzeppelin-eth/blob/master/contracts/token/ERC20/StandaloneERC20.sol) provided by the OpenZeppelin package. We will _initialize_ the instance with the token metadata (name, symbol, and decimals), and mint a large initial supply for one of our accounts.
+
+<!-- CODE: We need a command to retrieve the current accounts (#967) -->
+
+> Note: Your available accounts are shown by ganache when you start the process. If you ran it with the `--deterministic` flag as instructed, then your first and default account will be `0x90f8bf6a479f320ead074411a4b0e7944ea8c9c1`.
+
+```console
+$ npx zos create
+? Pick a contract to instantiate: openzeppelin-eth/StandaloneERC20
+? Pick a network: local
+✓ Deploying openzeppelin-eth dependency to network
+? Do you want to call a function on the instance after creating it?: Yes
```

Comment:
Can we make 'yes' in bold? 
As I was going through the doc and creating contracts and running them, Every time I  just entered 'No', I did this twice automatically while deploying this contract too... and got confused ..

Reply:
Yes! It's on our list, but we don't have an easy way to add bold in code snippets. We are migrating to a new doc site generator soon, and we'll tackle this then.